<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsMssqlNewConnectionBase</class>
 <widget class="QDialog" name="QgsMssqlNewConnectionBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>352</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create a New MSSQL connection</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="GroupBox1">
     <property name="title">
      <string>Connection Information</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="TextLabel1_2">
            <property name="text">
             <string>Name</string>
            </property>
            <property name="buddy">
             <cstring>txtName</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Provider/DSN</string>
            </property>
            <property name="buddy">
             <cstring>txtService</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel1">
            <property name="text">
             <string>Host</string>
            </property>
            <property name="buddy">
             <cstring>txtHost</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel2">
            <property name="text">
             <string>Database</string>
            </property>
            <property name="buddy">
             <cstring>txtDatabase</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel3">
            <property name="text">
             <string>Username</string>
            </property>
            <property name="buddy">
             <cstring>txtUsername</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel3_2">
            <property name="text">
             <string>Password</string>
            </property>
            <property name="buddy">
             <cstring>txtPassword</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="txtName">
            <property name="toolTip">
             <string>Name of the new connection</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtService"/>
          </item>
          <item>
           <widget class="QLineEdit" name="txtHost"/>
          </item>
          <item>
           <widget class="QLineEdit" name="txtDatabase"/>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_trustedConnection">
            <property name="text">
             <string>Trusted Connection</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtUsername"/>
          </item>
          <item>
           <widget class="QLineEdit" name="txtPassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_1">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="chkStoreUsername">
          <property name="text">
           <string>Save Username</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QPushButton" name="btnConnect">
          <property name="text">
           <string>&amp;Test Connect</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="chkStorePassword">
          <property name="text">
           <string>Save Password</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cb_geometryColumns">
        <property name="text">
         <string>Only look in the geometry_columns metadata table</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="cb_allowGeometrylessTables">
        <property name="text">
         <string>Also list tables with no geometry</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="cb_useEstimatedMetadata">
        <property name="text">
         <string>Use estimated table parameters</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtService</tabstop>
  <tabstop>txtHost</tabstop>
  <tabstop>txtDatabase</tabstop>
  <tabstop>txtUsername</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>chkStoreUsername</tabstop>
  <tabstop>chkStorePassword</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsMssqlNewConnectionBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsMssqlNewConnectionBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
